buku

```diff
@@ -722,7 +722,6 @@ class BukuDb:
 
         :param tag: tag to search
         '''
```

Comment:
Please leave the newline after function desc. We follow the convention in `Buku`.